stm32_projects/device_drivers/minmea.h

```diff
@@ -216,6 +216,11 @@ bool minmea_parse_zda(struct minmea_sentence_zda *frame, const char *sentence);
 int minmea_gettime(struct timespec *ts, const struct minmea_date *date,
                    const struct minmea_time *time_);
 
+/**
```

Comment:
We don't really have a great convention for third party code but because the minmea code we have is otherwise unchanged from its source (other than I think we ran formatting on it) I think it would be good to avoid modifying it. This could probably just move to a local helper in `gps.cpp` or somewhere else if you think there's a better place.